packages/components/ovh-shell/src/client/index.ts

```diff
@@ -61,8 +61,11 @@ export function initStandaloneClientApi(
   applications: Record<string, Application>,
 ) {
   const appConfig = applications[appId];
+
   if (!appConfig) {
-    throw new Error(`Unknown application '${appId}'`);
+    if (window.location.hostname !== 'localhost') {
+      throw new Error(`Unknown application '${appId}'`);
+    }
```

Comment:
**nitpick:** This can be combined with above `if` statement
`if (!appConfig && window.location.hostname !== 'localhost') {